gettsim/data/arbeitsl_geld_2.yaml

```diff
@@ -1,730 +1,517 @@
     en:
-  values:
-    2005-01-01:
-      value: 13000
-      note: G. v. 24.12.2003 BGBl. I S. 2954.
-    2006-08-01:
-      value: 16250
-      note: G. v. 20.07.2006 BGBl. I S. 1706.
-    2008-01-01:
-      value: 16500
-      note: G. v. 20.04.2007 BGBl. I S. 554.
-    2010-04-17:
-      value: 49500
-      note: G. v. 14.04.2010 BGBl. I S. 410.
+  2005-01-01:
+    scalar: 13000
+    note: G. v. 24.12.2003 BGBl. I S. 2954.
+  2006-08-01:
+    scalar: 16250
+    note: G. v. 20.07.2006 BGBl. I S. 1706.
+  2008-01-01:
+    scalar: 16500
+    note: G. v. 20.04.2007 BGBl. I S. 554.
+  2010-04-17:
+    scalar: 49500
+    note: G. v. 14.04.2010 BGBl. I S. 410.
+
 a2vog4:
   name:
     de: Vermögensobergrenze für geldwerte Ansprüche, nach 1963 geboren
     en: Upper limit allowance (monetary claims), born after 1963
   description:
     de: Vermögensobergrenze für geldwerte Ansprüche nach § 12 (2) Satz 2 Nr. 3 SGB II. Ab 01.01.2008 gilt für Personen der Jahrgänge nach 1963 ein abweichender Betrag definiert in § 12 (2) Satz 4 Nr. 3 SGB II (bis 31.12.2007 einheitliche Beträge für alle Jahrgänge).
     en:
-  values:
-    2005-01-01:
-      value: 13000
-      note: G. v. 24.12.2003 BGBl. I S. 2954.
-    2006-08-01:
-      value: 16250
-      note: G. v. 20.07.2006 BGBl. I S. 1706.
-    2008-01-01:
-      value: 16750
-      note: G. v. 20.04.2007 BGBl. I S. 554.
-    2010-04-17:
-      value: 50250
-      note: G. v. 14.04.2010 BGBl. I S. 410.
+  2005-01-01:
+    scalar: 13000
+    note: G. v. 24.12.2003 BGBl. I S. 2954.
+  2006-08-01:
+    scalar: 16250
+    note: G. v. 20.07.2006 BGBl. I S. 1706.
+  2008-01-01:
+    scalar: 16750
+    note: G. v. 20.04.2007 BGBl. I S. 554.
+  2010-04-17:
+    scalar: 50250
+    note: G. v. 14.04.2010 BGBl. I S. 410.
+
 a2we:
   name:
     de: Abzugsfähige Werbungskostenpauschale
     en: Allowance for expenses
   description:
     de: Abzugsfähige monatliche Werbungskostenpauschale. § 6 (1) Nr. 3 a) ALG II-V. 2005-01.01 bis 2016-07-31. Betrag mit V. v. 26.07.2016 BGBl. I S. 1858 entfallen.
     en:
-  values:
-    2005-01-01:
-      value: 15.33
-      note: V. v. 20.10.2004 BGBl. I S. 2622. Es gilt 1/60 der ein Sechzigstel der steuerrechtlichen Werbungskostenpauschale (Parameter werbung). Berechnung werbung/60 = 920/60.
-    2011-01-01:
-      value: 15.33
-      note: V. v. 21.06.2011 BGBl. I S. 1175. Fester Betrag. Betrag unverändert.
+  2005-01-01:
+    scalar: 15.33
+    note: V. v. 20.10.2004 BGBl. I S. 2622. Es gilt 1/60 der ein Sechzigstel der steuerrechtlichen Werbungskostenpauschale (Parameter werbung). Berechnung werbung/60 = 920/60.
+  2011-01-01:
+    scalar: 15.33
+    note: V. v. 21.06.2011 BGBl. I S. 1175. Fester Betrag. Betrag unverändert.
```

Comment:
I also added the question there why we have this, if it is a fixed value? And outdated means, we should set 0 here?

Reply:
In principle yes, because it will need to be calculated in a new function. And we'd need more parameters. For the time being, I'd say leave as is because it'll change tests as well.

Reply:
Then could please create an issue describing, exactly what needs to be fixed here? Then we don't forget!